server/stream.go

```diff
@@ -2991,7 +2991,7 @@ func (mset *stream) internalLoop() {
 					pm.o.didNotDeliver(pm.seq)
 				}
 			}
-			c.flushClients(10 * time.Millisecond)
+			c.flushClients(0)
```

Comment:
I think we should also look at the sendq here: https://github.com/nats-io/nats-server/blob/bd5d5d0ff89516c8702c9a0e31c004b8da315c63/server/sendq.go#L87

Reply:
Good call. Agree.

Reply:
ok done.